doc/faq.rst

```diff
@@ -348,17 +350,23 @@ as follows:
     Restart Salt Minion:
       cmd.run:
     {%- if grains['kernel'] == 'Windows' %}
-        - name: 'C:\salt\salt-call.bat --local service.restart salt-minion'
+        - name: 'C:\salt\salt-call.bat service.restart salt-minion'
     {%- else %}
-        - name: 'salt-call --local service.restart salt-minion'
+        - name: 'salt-call service.restart salt-minion'
     {%- endif %}
         - bg: True
         - onchanges:
           - pkg: Upgrade Salt Minion
 
 However, it requires more advanced tricks to upgrade from legacy version of
 Salt (before ``2016.3.0``) on UNIX-like operating systems, where executing
-commands in the background is not supported:
+commands in the background is not supported. You also may need to schedule
+restarting of the Minion service using :ref:`masterless mode
+<masterless-quickstart>` after all other states have been applied for Salt
+version earlier than ``2016.11.0``. This allows the minion to keep the Minion
```

Comment:
`version` --> `versions`

And I think this part:

> This allows the minion to keep the Minion

could be clearer. Maybe something like `This allows the Minion to keep the Minion-Master connection alive ...`

What do you think?

Reply:
Yep, that does not sound well. I will fix all these stuff. Thanks for reviewing!